apps/web/screens/UniversitySearch/Sitemap.tsx

```diff
@@ -0,0 +1,58 @@
+import { NextApiResponse } from 'next/types'
+import { NormalizedCacheObject } from '@apollo/client/cache/inmemory/types'
+import { ApolloClient } from '@apollo/client/core/ApolloClient'
+
+import { GetUniversityGatewayProgramsQuery } from '@island.is/web/graphql/schema'
+
+import { GET_UNIVERSITY_GATEWAY_PROGRAM_LIST_IDS } from '../queries/UniversityGateway'
+
+const Sitemap = () => null
+
+Sitemap.getProps = async ({
+  apolloClient,
+  res,
+  locale,
+}: {
+  apolloClient: ApolloClient<NormalizedCacheObject>
+  res: NextApiResponse
+  locale: string
+}) => {
+  const { data } = await apolloClient.query<GetUniversityGatewayProgramsQuery>({
+    query: GET_UNIVERSITY_GATEWAY_PROGRAM_LIST_IDS,
+  })
+
+  const baseUrl =
+    locale === 'is'
+      ? 'https://island.is/haskolanam/'
```

Comment:
I guess it doesn't matter if we hardcode island.is here but just wanted to point out that it could be possible to set the hostname depending on the environment (beta.dev01.devland.is for dev and so forth).

This was done here: apps/web/pages/api/rss/index.ts if you want to check that out